src/main/kotlin/hu/kirdev/schpincer/model/CircleEntity.kt

```diff
@@ -79,6 +79,9 @@ data class CircleEntity(
         @Column
         var virGroupId: Long? = null
 
+        @Column
```

Comment:
It requires a column definistion with '... default 0' parameter in order to make DB migration automatic

Reply:
I've found this: https://stackoverflow.com/a/42150940 as an alternative to this: https://stackoverflow.com/a/375202
I think you were mentioning the latter.

However, can you check whether the former is valid? It seems a better solution, as you already use `org.hibernate`.

Reply:
I've implemented the one using `org.hibernate`'s `ColumnDefault` for now